src/Control/Apply.js

```diff
@@ -1,6 +1,6 @@
-"use strict";
 
-exports.arrayApply = function (fs) {
```

Comment:
My `sed` skills aren't good enough to know how to delete the `"use strict"` line followed by any empty lines. If someone else can provide a solution I can use in my script, let me know.

Reply:
Ok, I reverted to using a node script to handle the multi-line replacement.